src/server/pkg/deploy/assets/assets.go

```diff
@@ -227,6 +227,11 @@ func replicas(r int32) *int32 {
 	return &r
 }
 
+// Kubernetes doesn't work well with windows path separators
+func kubeFilepathJoin(paths ...string) string {
+	return strings.ReplaceAll(filepath.Join(paths...), "\\", "/")
+}
```

Comment:
This will still be fine on linux system right? I assume CI will tell us that since travis is linux.

Reply:
Yeah, I'll add a windows check around this =(
Good catch!